sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1779,6 +1791,81 @@ public static CaseDataDto toDto(Case source) {
 		return target;
 	}
 
+	@Override
+	public CaseDataDto convertContactToCase(ContactDto contact, PersonReferenceDto person, Disease disease) {
+		return convertContactToCase(new CaseDataDto(), contact, person, disease, null);
+	}
+
+	@Override
+	public CaseDataDto convertContactToCase(ContactDto contact, PersonReferenceDto person, Disease disease,
+			EventParticipantDto eventParticipant) {
+		return convertContactToCase(new CaseDataDto(), contact, person, disease, eventParticipant);
+	}
+
+	@Override
+	public CaseDataDto convertContactToCase(CaseDataDto caseData, ContactDto contact, PersonReferenceDto person,
```

Comment:
Will the  **buildFromEventParticipant(EventParticipantDto eventParticipant, Disease eventDisease)** method fill symptoms to? 
if yes should the method use `VisitFacade.getLastVisitByPerson(person, disease, date)` to get the last visit of the `eventParticipant.getPerson()` use it further to get the last visit symptoms?

Reply:
@opatajoshua Good question. The answer is no, since event participants won't have a visit. Visits will only be created in the context of a contact. 

Background: We would have preferred to only associate a visit with a contact and not a person. Unfortunately it's possible that a person is multiple contacts at the same time, because the person came close to multiple cases in real life. In this scenario the visits need to be shared for all contacts of the person. That's why they are only associated to the person.

Maybe we should change this and have a direct association between a visit and multiple contacts - that would make it more clear.

Reply:
alright